src/app/static/src/tests/utils.test.ts

```diff
@@ -190,14 +190,6 @@ describe("utils", () => {
                     url: "http://other",
                     name: "Other Resource"
                 },
-                {
-                    resource_type: "test-type",
-                    id: "456",
-                    name: "A different resource",
-                    last_modified: "2021-03-01",
-                    metadata_modified: "2021-02-28",
-                    url: "http://test2"
-                },
                 {
                     resource_type: "test-type",
                     id: "123",
```

Comment:
I should probably have passed a different resource name argument to this test, just to show it wasn't being overwritten in this case.

Reply:
Yes, that would be worth testing